### DISCUSSION TITLE
Is it possible to use pmon/supervisor only?

### DISCUSSION THREAD

**Discussion by Amer-shanawany** [troglobit/watchdogd#35] [Q&A] [answered]
Hi,

I'm working on an embedded device, and currently looking for a replacement to *monit*.
We have a script that kicks the watchdog, upon checking *monit* status.
Is it possible to use *watchdogd* only as process monitor and not to interact with the kernel by enabling test mode  *TESTMODE_DISABLED* and using *-S* argument?

**Comment by troglobit** [comment, marked as answer]
No. That's not what watchdogd is designed for.  It is designed to monitor a heartbeat of services and then acknowledge to hardware that the software is indeed still alive and kicking.

**Reply by Amer-shanawany** [reply]
Thanks for the fast reply.

**Reply by troglobit** [reply]
np :-)
